extensions/replication/tasks/MultipleBackendTask.js

```diff
@@ -132,6 +132,35 @@ class MultipleBackendTask extends ReplicateObject {
         });
     }
 
+    /**
+     * Used when an operation performed in the process of replicating a
+     * multipart upload fails.
+     * @param {ObjectQueueEntry} sourceEntry - The source object entry
+     * @param {String} uploadId - The upload ID of the initiated MPU
+     * @param {Werelogs} log - The logger instance
+     * @param {Function} cb - The callback to call
+     * @return {undefined}
+     */
+    _handleMPUFailure(sourceEntry, uploadId, log, cb) {
+        this._multipleBackendAbortMPU(sourceEntry, uploadId, log, err => {
+            if (err) {
+                log.error('an error occurred when aborting a MPU', {
+                    method: 'MultipleBackendTask._handleMPUFailure',
+                    entry: sourceEntry.getLogInfo(),
+                    origin: 'target',
+                    peer: this.destBackbeatHost,
+                    error: err.message,
+                });
+                return cb(err);
+            }
+            log.info('successfully aborted a MPU', {
```

Comment:
very minor but easy change: when an operator sees "successfully" he may think the operation was a success, I would rather simply state "aborted MPU" so it's still clear something was wrong at the beginning.

Reply:
Good point—"success" is probably an overstatement of had happened here.